packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -5,11 +5,135 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../globals/imports';
-
 @mixin leadspace {
   .#{$prefix}--leadspace {
-    background-color: transparent;
+    position: relative;
+    padding-top: carbon--rem(370px) / carbon--mini-units(40) * 100%;
+    height: 0;
+    background: $ui-background;
+
+    &__image {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    &__container {
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+    }
+
+    &__overlay {
+      padding: $layout-03 $layout-01;
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+      background-image: linear-gradient(
+        to bottom,
+        rgba($ui-background, 1),
+        transparent
+      );
+    }
+
+    &__content {
+      position: absolute;
+      bottom: $layout-03;
+    }
+
+    h1 {
+      @include carbon--type-style(display-01, true);
+      color: $text-01;
+    }
+
+    p {
+      @include carbon--type-style(expressive-heading-03, true);
+      color: $text-01;
+    }
+
+    &__ctas {
+      display: flex;
+      flex-direction: column-reverse;
+    }
+
+    button,
+    a {
+      width: 100%;
+    }
+
+    button:nth-child(2),
+    a:nth-child(2) {
+      margin: $layout-03 0 $layout-01 0;
+    }
+  }
+
+  .#{$prefix}--leadspace--productive h1 {
+    @include carbon--type-style(expressive-heading-06, true);
+  }
+
+  @include carbon--breakpoint(md) {
+    .#{$prefix}--leadspace {
+      padding-top: carbon--mini-units(50) / carbon--mini-units(84) * 100%;
+
+      &__overlay {
+        padding: $layout-03;
+        background-image: linear-gradient(
+          to right,
+          rgba($ui-background, 1),
+          transparent
+        );
+        background-size: 50%;
+        background-repeat: no-repeat;
+      }
+
+      &__content {
+        width: 100%;
+      }
+
+      h1 {
+        @include carbon--make-col(7, 8);
+      }
+
+      p {
+        width: 50%;
+      }
+
+      &__ctas {
+        flex-direction: row-reverse;
+        float: left;
+        margin-top: $layout-03;
+      }
+
+      button,
+      a {
+        width: auto;
+      }
+
+      button:nth-child(2),
+      a:nth-child(2) {
+        margin: 0 $layout-01 0 0;
+      }
+    }
+  }
+
+  @include carbon--breakpoint(lg) {
+    .#{$prefix}--leadspace {
+      padding-top: carbon--mini-units(60) / carbon--mini-units(132) * 100%;
+
+      h1 {
+        @include carbon--make-col(8, 16);
```

Comment:
I still don't think this grid is right. You need the `carbon--make-container`, `carbon--make-row`, and `carbon--make-col-ready`.

![image](https://user-images.githubusercontent.com/3793636/64877259-a35e4680-d616-11e9-8b14-926e3212ac88.png)

Reply:
I initially added those and I believe the padding was weird but didn't dig further - let me try again and see if I'm getting any issues.